src/containers/Review/assignment/AssignmentSectionRow.tsx

```diff
@@ -122,9 +125,20 @@ const AssignmentSectionRow: React.FC<AssignmentSectionRowProps> = ({
     ({ text }) => text != strings.ASSIGNMENT_YOURSELF
   )
 
+  const levelName =
+    structure.stages
+      .find(({ stage: { number } }) => number === structure.info.current.stage.number)
+      ?.levels.find(({ number }) => reviewLevel === number)?.name || strings.LEVEL_NOT_FOUND
+
   return (
-    <Grid className="section-single-row-box-container">
+    <Grid columns={2} className="section-single-row-box-container">
       <Grid.Row>
+        <Grid.Column className="review-level">
+          <Label
+            className="simple-label"
+            content={`${strings.REVIEW_FILTER_LEVEL}: ${levelName}`}
+          />
+        </Grid.Column>
         <Grid.Column className="centered-flex-box-row">
```

Comment:
```suggestion
    <Grid columns={2} stackable className="section-single-row-box-container">
      <Grid.Row>
        <Grid.Column className="review-level" width={5}>
          <Label className="simple-label">
            {strings.REVIEW_FILTER_LEVEL}:&nbsp;&nbsp;<strong>{levelName}</strong>
          </Label>
        </Grid.Column>
        <Grid.Column className="centered-flex-box-row" width={11}>
```
I think these proportions are a little nicer. (FYI I told you wrong, it's the "width" prop you needed, "colspan" is just HTML)
Also, **bolding** of the Level name for consistency with my comment on the last PR (bold the review name)